src/app/teams/teams.component.ts

```diff
@@ -47,6 +46,8 @@ export class TeamsComponent implements OnInit, AfterViewInit {
   deleteDialog: any;
   readonly myTeamsFilter = this.route.snapshot.data.myTeams ? 'on' : 'off';
   mode: 'team' | 'enterprise' = this.route.snapshot.data.mode || 'team';
+  displayedColumns = this.planetType === 'community' && this.mode === 'enterprise' ?
```

Comment:
This is a change from #4776.  Please make sure that when you create a new branch you are on the master branch (`git checkout master`)